src/config_creature.c

```diff
@@ -76,6 +76,8 @@ const struct NamedCommand creaturetype_experience_commands[] = {
   {NULL,                           0},
   };
 
+// It is pointless to use all uppercase letters here.
```

Comment:
While this is true, some consistency is usually nice. We could change the entire list to CamelCase?

Reply:
I don't object to your proposal.